server/graphql/schema/asset/resolvers/mutations/uploadAsset.ts

```diff
@@ -70,11 +70,10 @@ class UploadAssetWorker extends ResolverBase {
             LOG.error('uploadAsset unable to retrieve user context', LOG.LS.eGQL);
             return { status: UploadStatus.Noauth, error: 'User not authenticated' };
         }
-
-        if (this.idSOAttachment)
-            await this.appendToWFReport(`<b>Upload starting</b>: ATTACH ${filename}`, true);
-        else if (!this.idAsset)
+        if (this.idAsset)
             await this.appendToWFReport(`<b>Upload starting</b>: ADD ${filename}`, true);
```

Comment:
I believe you translated this logic incorrectly and introduced a minor bug.  When `this.idAsset` is truthy, aren't we in "UPDATE" mode?  Please correct the logic here -- "ADD" should be "UPDATE", and below, "UPDATE" should be "ADD"